Sockets/BackEndMonitoringSockets/include/CAcceptorWrapper.h

```diff
@@ -26,14 +26,16 @@ class CAcceptorWrapper
 	bool StopSocket();
 
 private:
-	const int c_max_idle_iteration = 60000;
 	void InitAcceptor(int port, const std::string& address);
 	void InitServiceHandler(CDataReceiver& json_data);
 	void InitSocketWrapper();
 	bool HandleEvents();
-	void AddClientToThread(int socket_fd);
+	bool AcceptRequest();
+	void AddClientToThread(int socket_fd, bool& is_broken_socket);
```

Comment:
Maybe it will be better that this method would return bool, instead of passing it by ref

Reply:
Now the method returns bool